app/views/layouts/_organiser_menu.html.haml

```diff
@@ -15,3 +15,4 @@
     %label My Chapters
   - current_user.organised_chapters.each do |chapter|
     %li= link_to chapter.name, admin_chapter_path(chapter)
+    %li= link_to 'Feedback', admin_chapter_feedbacks_path(chapter)
```

Comment:
This is a minor issue but I'm wondering if it would be better to move this link into the organiser chapter view - if a person organises more than 1 chapters these links might become confusing (e.g. there would be two Feedback links). Or add the chapter name to the link text.